docs/ops-internal/checking-for-config-changes.md

```diff
@@ -0,0 +1,219 @@
+# Checking for config changes
+
+The rpc-ceph project provides the ability to test for configuration differences between rpc-ceph and a running ceph-cluster
+that was previously deployed by either ceph-ansible or another version of rpc-ceph.  Once you have bootstrapped rpc-ceph
+and configured your variable overrides to match the values of your running cluster, you can run the
+`check-for-config-changes.yml` playbook to check and see if any values will change if you were to apply rpc-ceph.
+
+## Running check-for-config-changes.yml
+
+```
+ceph-ansible-playbook -i <inventory> playbooks/check-for-config-changes.yml
+```
+
+## Ceph Config Changes
+
+For each node in your ceph cluster a new ceph configuration will be generated in a temporary location and compared
+against your current ceph config file.  The result will be a dict of config that has been 'ADDED' in the new config,
+config that has 'CHANGED' from the old config to the new config and config that has been 'REMOVED' in the new config.
+
+### ADDED Config
+
+Config that has been 'added' is config that you were not setting before and were just taking the default.  The
+assumption here is that the new config value is different from the default value.  For each added config you will
+need to decide whether you want to accept the new value or update your variables files to use the old value. To
+unset one of rpc-ceph's default values and continue to use Ceph's default value you will want to add the
+configuration setting to `ceph_conf_overrides_extra` dict with a value of `default(omit)`.
+
+### CHANGED Config
+
+Config that has 'changed' will be listed with what the 'current value' is accroding to the ceph.conf file and what the
+'new value' will be when rpc-ceph is applied.  You can either accpet the new value or update the variables file to
```

Comment:
We should probably clarify that accepting the config file means just running the regular deploy-ceph.yml playbook and the changes will be made (including service restarts).

Reply:
can do